src/Analysis/GWASUIApp/GWASUIApp.jsx

```diff
@@ -26,46 +24,63 @@ const GWASUIApp = (props) => {
   }, [props]);
 
   if (!gwasTypeSelected) {
+    const isEnterOrSpace = (event) => (event.key === 'Enter' || event.key === ' ' || event.key === 'Spacebar'
+      || event.keycode === '32' || event.keycode === '13');
```

Comment:
why are both key and keycode needed?

Reply:
Its complicated: https://javascript.info/keyboard-events